src/services/expenses.js

```diff
@@ -0,0 +1,80 @@
+'use strict';
+
+let expenses = [];
+
+function init() {
+  expenses = [];
+};
+
+function getAll(userId, category, from, to) {
+  if (category) {
+    const filteredExpenses = expenses
+      .filter(expense => expense.category === category);
+
+    return filteredExpenses;
+  }
+
+  if (from && to) {
+    const filteredExpenses = expenses
+      .filter(expense =>
+        expense.spentAt >= from && expense.spentAt <= to,
+      );
+
+    return filteredExpenses;
+  }
+
+  if (userId) {
+    const foundUser = expenses
+      .filter(expense => expense.userId === +userId);
+
+    return foundUser;
+  }
+
+  return expenses;
+};
+
+function getbyId(expenseId) {
+  const foundExpense = expenses.find(expense => expense.id === +expenseId);
+
+  return foundExpense || null;
+}
+
+function create(userId, spentAt, title, amount, category, note) {
```

Comment:
pass the whole body as a param, do you really need to describe each field individually?